technical-reports/Performance-Testing.md

```diff
@@ -0,0 +1,29 @@
+# Performance Testing of TEAMMATES
+
+This report gives a detailed explanation of the profiling operations performed on TEAMMATES. It gives an outline of the
```

Comment:
`describes the reasons behind our proposed solution` instead?
I feel that this is the main purpose of the report. Of course, we can still discuss what parts our solution consists of as well (in the form of a workflow diagram perhaps?).

The actual solution is kinda described in the performance-testing docs too.